manager/container.go

```diff
@@ -270,6 +273,44 @@ func (c *containerData) updateLoad(newLoad uint64) {
 	}
 }
 
+func (c *containerData) updateSubcontainerStats(stats *info.ContainerStats, path string) (*info.ContainerStats, error) {
```

Comment:
Ah, I see what this is doing. In my mind, I was assuming that both load and task stats will be hierarchically reported.
This approach works too, but I am a little worried if we are doing repetitive work unnecessarily. We might also be maintained a large list of subcontainers for root, even though some of them are empty.

Can you measure the amount of work we end up doing for a busy machine? We might be able to make the current solution a bit more efficient if required.